zsh/.zshrc.d/prompt

```diff
@@ -1,5 +1,6 @@
 #!/bin/zsh
 # shellcheck shell=bash
+# shellcheck disable=SC2034
```

Comment:
Don't disable this globally.  Just disable it on the lines where it is needed.

Reply:
I'll remove it entirely now that I've excluded the `zsh` files.

Reply:
Fixed in https://github.com/cisagov/.dotfiles/pull/2/commits/e4d2a6d4a48c1b66d6ff25dfc8baf93d808fe876